### PR TITLE
LPS-168431 Accessibility collections variations: when a variation is prioritized or deprioritized  the alert message should be more clear

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://issues.liferay.com/browse/LPS-168431)

## Steps

1. Go to People -> Segments and create one segment.
3. Go to Site Builder -> Collections and create one manual collection of All types
4. Create one variation for the collection and prioritize it

Screenshot 2023-03-31 at 14 40 30

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
Just started reviewing :)

**Review by claraizquierdo — COMMENTED** [review]
Hi @DiegoHu97 ,

As we talk via Slack, it would be great to apply the accessibility improvements to the error toast too ❤️

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -19029,6 +19029,8 @@ variants=Variants
 variants-available=Variants Available
 variants-help=Create at least one variant to run the test.
 variation=Variation
+variation-priority-changed-successfully=Variation priority changed successfully
```
Comment (claraizquierdo) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:19032]: Hi @DiegoHu97, everything looks fine except for  like this translation. I think it is not necessary.

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Hi @DiegoHu97 ,
I just left one comment. Also, do you mind updating the description of the bug in the jira ticket so that the QA team knows what to test? Thank you ❤️

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (DiegoHu97) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: Hey @claraizquierdo, I have updated the PR. Thanks!

**Comment by DiegoHu97** [comment]
ci:test:sf

**Review by claraizquierdo — APPROVED** [review]
LGTM! 🎉

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:reevaluate:1327401_8912

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:reevaluate:1257401_400

**Comment by claraizquierdo** [comment]
@DiegoHu97 , looks like the frontend unit tests might be failing.

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/132946

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**